server-sdk/src/test/java/com/amazonaws/ml/mms/servingsdk/ModelServerEndpointTest.java

```diff
@@ -0,0 +1,38 @@
+package com.amazonaws.ml.mms.servingsdk;
+
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+/**
+ * Unit test for simple App.
+ */
+public class ModelServerEndpointTest
+    extends TestCase
+{
+    /**
+     * Create the test case
+     *
+     * @param testName name of the test case
+     */
+    public ModelServerEndpointTest(String testName )
+    {
+        super( testName );
+    }
+
+    /**
+     * @return the suite of tests being tested
+     */
+    public static Test suite()
+    {
+        return new TestSuite( ModelServerEndpointTest.class );
+    }
+
+    /**
+     * Rigourous Test :-)
```

Comment:
I like the comment on this one :)

Reply:
Auto generated :P .. I will need to redo the tests . This is only to get the Maven package work progressing ..